src/ILRepack.props

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <ILRepackEnabled>true</ILRepackEnabled>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(ILRepackEnabled)' == 'true' ">
+    <!-- Make sure the types of the API between plug-in and MsBuild in in `Exclude`. CODESYNC[DO_NOT_ILMERGE_ASSEMBLIES] -->
+    <PluginAssemblies Include="System.Collections.Immutable.dll" />
+    <PluginAssemblies Include="Microsoft.Build.dll" />
+    <PluginAssemblies Include="Microsoft.Build.Framework.dll" />
+    <PluginAssemblies Include="Microsoft.Build.Utilities.Core.dll" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(ILRepackEnabled)' == 'true' ">
+    <PackageReference Include="ILRepack">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+
+    <!-- BuildXL.Cache.ContentStore.Distributed.dll depends on this but doesn't list it-->
```

Comment:
Can we get the Cache team to fix this?

Reply:
ok i fixed this by excluding BuildXL.Cache.ContentStore.Distributed.dll

Reply:
update: nevermind - that didn't work